html/pages/device/nfsen.inc.php

```diff
@@ -1,16 +1,53 @@
 <?php
 
-$datas = array(
-          'Flows'   => 'nfsen_flows',
-          'Packets' => 'nfsen_packets',
-          'Traffic' => 'nfsen_traffic',
-         );
+print_optionbar_start();
 
-foreach ($datas as $name => $type) {
-    $graph_title = $name;
-    $graph_type  = 'device_'.$type;
+$link_array = array(
+    'page'   => 'device',
+    'device' => $device['device_id'],
+    'tab'    => 'nfsen',
+    );
 
-    include 'includes/print-device-graph.php';
+echo generate_link('General', $link_array, array('nfsen' => 'general'));
+
+$printedChannel='0';
```

Comment:
Using `false` and `true` for this variable makes it more obvious what it is for.